pkg/setting/setting.go

```diff
@@ -1591,6 +1592,7 @@ func readAuthSettings(iniFile *ini.File, cfg *Cfg) (err error) {
 	// basic auth
 	authBasic := iniFile.Section("auth.basic")
 	cfg.BasicAuthEnabled = authBasic.Key("enabled").MustBool(true)
+	cfg.BasicAuthStrongPasswordPolicy = authBasic.Key("strong_password_policy").MustBool(false)
```

Comment:
Please either update here or in the `defaults.ini` and `sample.ini`, because the `ini` files contain `password_policy` as a key.

Reply:
This was a mistake by my part, I have fixed this